config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  get 'users/index'
+  get 'users/show'
+  get 'posts/index'
+  get 'posts/show'
+  get 'users/index'
```

Comment:
- [ ] I would recommend removing all the routes from lines 2 to 6. As we have mentioned the routes for user and post in lines 9 to 11 which is a more RESTFUL way to achieve the desired URLs mentioned in the project requirements. When I run `rails routes` the result of the URLs I get is extra what we needed as per the project requirements. See below:

URLs from project requirements | URLs from project
:---:|:---:
![](https://user-images.githubusercontent.com/45002654/149008687-e33ca00d-89df-490d-b203-601bc363d73f.png) | ![Screenshot from 2022-01-13 15-14-48](https://user-images.githubusercontent.com/22434660/149306602-a886b390-f147-4ea3-9d7a-b20d27ada4e3.png)

Reply:
Sorry 😞 I have forgotten to remove those lines, thank you for the suggestion !!